packages/cli/src/cli.ts

```diff
@@ -101,6 +107,10 @@ function readScanArguments(): ScanArguments {
                 if (isEmpty(args.url)) {
                     throw new Error('The --url option is required.');
                 }
+
+                if (args.restart === true && args.continue === true) {
```

Comment:
Seems like we don't have tests for this or the cli-client-runner changes. Is this intentional/necessary?

Reply:
will add later